src/hardwareModel.py

```diff
@@ -160,6 +163,9 @@ def init_memory(self, mem_needed, nvm_mem_needed):
         ).items():
             data["var"] = ""  # reg keeps track of which variable it is allocated
 
+        self.mem_size = mem_needed
+        self.bus_width = 256 # ask if want as input
```

Comment:
read this from hw_cfsg.ini for now. This will become a parameter that we tune i nthe simulator later.
But for now just read it. See above.